test/unit/spec/PasswordExpired_spec.js

```diff
@@ -24,8 +24,12 @@ function (Q, _, $, OktaAuth, LoginUtil, Util, PasswordExpiredForm, Beacon, Expec
   var itp = Expect.itp;
   var tick = Expect.tick;
   var processCredsSpy = jasmine.createSpy('processCredsSpy');
```

Comment:
Hmm, strange, not sure why we made these global - feels like this might actually result in hard-to-track bugs if we we didn't reset the spy calls between tests. 

Instead, looking at the test code, feels like we should just define spies for processCreds when we're actually doing a processCreds type test.

For example, in setup, we could add this code:
```javascript
// This demonstrates the new code to add
function setup(settings, res) {
  // convenience so you don't have to do checks for existince later
  settings || (settings = {});

  // Also need a successSpy and use it instead of tick's below
  var successSpy = jasmine.createSpy('successSpy');

  var router = new Router({
    globalSuccessFn: successSpy,
    processCreds: settings.processCreds
    // other stuff
  });

  return Expect.waitForPasswordExpired({
    // other stuff
    successSpy: successSpy
  });
}
```

And then the test would look like this:
```javascript
itp('calls async processCreds function before saving a model', function () {
  const processCredsSpy = jasmine.createSpy().and.callFake(function (creds, cb) {
    cb();
  });
  return setup({ processCreds: processCredsSpy })
  .then(function (test) {
    test.setNextResponse(resSuccess);
    submitNewPass(test, 'oldpwd', 'newpwd', 'newpwd');
    return Expect.waitForSpyCall(test.successSpy, test);
  })
  .then(function (test) {
    // We know if we got here we invoked the success callback
    expect(processCredsSpy.calls.count()).toBe(1);
    expect(processCredsSpy).toHaveBeenCalledWith({
      username: 'inca@clouditude.net',
      password: 'newpwd'
    }, jasmine.any(Function));
  })
});
```